src/aspire/source/coordinates.py

```diff
@@ -46,7 +47,17 @@ class CoordinateSource(ImageSource, ABC):
     This also allows the CoordinateSource to be saved to an `.mrcs` stack.
     """
 
-    def __init__(self, files, particle_size, max_rows, B, symmetry_group):
+    def __init__(
+        self, files, particle_size, max_rows, B, symmetry_group, pixel_size=None
+    ):
+        """
+        :param files: A list of tuples of the form (path_to_mrc, path_to_coord)
+        :param particle_size: Desired size of cropped particles (will override the size specified in coordinate file)
+        :param max_rows: Maximum number of particles to read. (If `None`, will attempt to load all particles)
+        :param B: CTF envelope decay factor
+        :param symmetry_group: A `SymmetryGroup` object or string corresponding to the symmetry of the molecule.
+        :param pixel_size: Pixel size of the images in angstroms, default `None`.
```

Comment:
We should indicate what `None` means in this context (auto-assign from CTF if possible).

Reply:
Sure I changed the three occurrences to the following

```
        :param pixel_size: Pixel size of the images in angstroms.               
            Default `None` will attempt to infer `pixel_size` from              
            `CTFFilter` objects when available.  
            
```